From 24a8ef05a123c1fb7999afb6e25ed25b4ddefece Mon Sep 17 00:00:00 2001
From: Naveen M K <naveen521kk@gmail.com>
Date: Fri, 23 Jun 2023 20:13:23 +0530
Subject: [PATCH 061/N] configure: always build _multiprocessing on Windows

sem_unlink is not needed on Windows for the module.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index dd855db..1fa8479 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8320,7 +8320,7 @@ PY_STDLIB_MOD_SIMPLE([_zoneinfo])
 
 dnl multiprocessing modules
 PY_STDLIB_MOD([_multiprocessing],
-  [], [test "$ac_cv_func_sem_unlink" = "yes"],
+  [], [test "$ac_cv_func_sem_unlink" = "yes" -o "$MACHDEP" = "win32"],
   [-I\$(srcdir)/Modules/_multiprocessing])
 PY_STDLIB_MOD([_posixshmem],
   [], [test "$have_posix_shmem" = "yes"],
